patches/chrome-browser-browsing_data-chrome_browsing_data_remover_delegate_factory.cc.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.cc b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.cc
+index 794745b946a57e6aaed1deb7896edcb8320a8e06..5881b1d96414fd3e925100b7c8561154108d1fcd 100644
+--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.cc
++++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.cc
+@@ -5,6 +5,7 @@
+ #include "chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.h"
+ 
+ #include "base/memory/singleton.h"
++#include "brave/browser/rewards_data/brave_rewards_data_remover_delegate.h"
```

Comment:
not sure why this is marked as resolved because it's still here

Reply:
Forgot to run `update_patches`. Done.